x-pack/plugin/src/test/resources/rest-api-spec/api/async_search.get.json

```diff
@@ -0,0 +1,42 @@
+{
+  "async_search.get":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/async-search.html"
+    },
+    "stability":"experimental",
+    "url":{
+      "paths":[
+        {
+          "path":"/_async_search/{id}",
+          "methods":[
+            "GET"
+          ],
+          "parts":{
+            "id":{
+              "type":"string",
+              "description":"The async search ID"
+            }
+          }
+        }
+      ]
+    },
+    "params":{
+      "wait_for_completion":{
+        "type":"time",
+        "description":"Specify the time that the request should block waiting for the final response (default: 1s)"
+      },
+      "last_version":{
```

Comment:
another small which we already discussed in another channel: I find naming not so clear for this parameter. If we do need it, I would love to discuss options for naming it differently as the version terminology may be confusing for users. I would love to find something more self explanatory regarding the behaviour and the use for this.